src/configuration-factory.ts

```diff
@@ -30,18 +37,24 @@ export function configurationStorageFactory(
   if (forceMemoryOnly) {
     return new MemoryOnlyConfigurationStore();
   } else if (persistentStore) {
-    return new HybridConfigurationStore(new MemoryStore<Flag>(), persistentStore);
+    return new IsolatedHybridConfigurationStore(
```

Comment:
Introduced an `IsolatedHybridConfigurationStore` to allow isolating updates to the serving store from updates of the persistent store--useful in situations where we want per-session consistency.